src/User.php

```diff
@@ -5965,7 +5965,7 @@ private function showLdapInformation(): void
                 $info = AuthLDAP::getUserByDn(
                     $ds,
                     $this->fields['user_dn'],
-                    ['*', 'createTimeStamp', 'modifyTimestamp']
+                    ['*', '+', 'createTimeStamp', 'modifyTimestamp']
```

Comment:
1. We could add some documentation here
2. I just test with openldap and `createTimeStamp`/`modifyTimestamp` are already returned by the `+`  as they are operational attributes. I guess it is the same for all LDAPs servers.

```suggestion
                    [
                        // see https://docs.ldap.com/ldap-sdk/docs/tool-usages/ldapsearch.html
                        '*', // all user attributes
                        '+', // all operational attributes
                    ]
```